.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] <b>REQUIRED: </b> As requested by the previous reviewer, `node_modules` should not be committed as it contains the files that we don't need to push to our remote repo since those files will be generated every time we run `npm install` or `yarn install`. to fix this issue follow these simple steps.

   - Go to your root directory and run the following commands in order
      
      - `rm -r node_modules` or `sudo rm -r node_modules` in case the first one doesn't work due to permission issues.
      - `echo /node_modules  > .gitignore`
      - `npm install` or `yarn install` in case you are using yarn.
      - Then run `git status` and make sure that node_modules is not in changes history

Reply:
It's already inside .gitignore, dude. Like, literally, that's why it's green - cause I added it inside.